### ISSUE TITLE
The future of spacefish ✨🚀

### ISSUE THREAD

**Issue by matchai** [matchai/spacefish#187]
**TL;DR:** The team has begun working on the better, faster, cross-shell rewrite at https://github.com/starship/starship. ✨🚀
Spacefish is not going anywhere and is still accepting feature PRs, but we won't be actively working on it.

---

Inspired by the work @denysdovhan did on [denysdovhan/robbyrussell-node](https://github.com/denysdovhan/robbyrussell-node) and [this discussion](https://github.com/denysdovhan/spaceship-prompt/pull/571#issuecomment-441999293) back in November, I started work on https://github.com/starship/starship at the start of April 2019 to address the amount of duplicated effort that is made to maintain both spaceship and spacefish.

Starship is written in Rust and will contain all the logic required for a wrapper to be created for any shell. The plan for the immediate future is to add first-class support for Fish, Zsh, and Bash. The prompts will mostly be the same, with some improvements where it was difficult to make them in the past. Things like prompt configuration files, improved testing, self-updating, more safety when things go wrong, etc.

All of that being said, we will still be reviewing feature PRs on this repo for the next little while, so we haven't gone into full "maintenance mode". That won't be happening until starship has all the features and tooling needed to replace spacefish.

We would love to get the support of those of you who are fans of spacefish and would like to contribute to our Rust codebase.

Hope to see you over at https://github.com/starship/starship! 😄

**Comment by dm17** [comment]
@matchai Any reason to still use spacefish over starship? Tnx.

**Comment by matchai** [comment]
> @matchai Any reason to still use spacefish over starship? Tnx.

Nope, not anymore. Starship has now surpassed spacefish in terms of features and performance.

**Comment by IlanCosman** [comment]
@matchai Perhaps it's time to archive Spacefish? There hasn't been any activity here, and feature PRs haven't been getting reviewed (I don't blame you, you've been doing great work on Starship 😄). Thus, it would be more honest to archive the repository, and as you said, Starship has surpassed Spacefish in every regard.

**Comment by matchai** [comment]
Agreed. See y'all over on the Starship repo! 👋 🚀
